cmd/falco/resolver.go

```diff
@@ -100,13 +100,17 @@ func (f *FileResolver) MainVCL() (*VCL, error) {
 }
 
 func (f *FileResolver) Resolve(module string) (*VCL, error) {
+	module_path_with_extension := module
```

Comment:
Tiny suggestion: the variable name should be lower-camel case.

```suggestion
	modulePathWithExtension := module
```

Reply:
Yeah I was writing this at the same time as I was writing some python code and my brain got all python mode

Reply:
fixed